src/edu/csus/ecs/pc2/core/report/CLICSAwardsReport.java

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.report;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+import edu.csus.ecs.pc2.VersionInfo;
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.imports.clics.CLICSAward;
+import edu.csus.ecs.pc2.core.imports.clics.CLICSJsonUtilities;
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.model.Filter;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+
+
+/**
+ * Print CLICS Awards JSON
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class CLICSAwardsReport implements IReport {
+
+    private static final long serialVersionUID = 6270072776103982061L;
```

Comment:
This is just a comment, not a request for a change... but:  I wish someone would explain to me what sense there is in assigning a serialVersionUID like 6270072776103982061.   Random sVUID values seem pointless to me; I cannot think of a single instance where a human reading the code could make sense of them.   It seems much more logical to me to assign a sVUID of "1" to the first version, and then increment it each time a breaking change occurs in the class.

Reply:
https://docs.oracle.com/javase/6/docs/platform/serialization/spec/class.html#4100

Reply:
Thanks for that reference.  I do now at least see why the use of a long auto-generated sVUID is useful -- it encapsulates into the sVUID a number of characteristics of the class, rather than simply being a "version number".

What I still _don't_ see is how it is useful _in our situation_.  Once the sVUID is auto-generated, _it remains the same no matter what changes are made to the class_ until/unless it is regenerated (which I seriously doubt anyone is ever doing).  Further, the whole point is supposed to be to allow run-time detection of an attempt to use an incompatible class version -- which can only be detected if one first defines/decides/declares what is "compatible" or "incompatible".   This in turn requires a lot of subtle analysis of the potential effects of changes to the class -- again, something I seriously doubt any of our developers put much if any time into doing.

Can anyone point to even one time where they have regenerated the hard-coded sVUID in a class because they've made backwards-incompatible changes?   (Note that Eclipse doesn't regenerate those numbers automatically; you have to explicitly tell it to do that.  I'm basically arguing that doing that is more or less the same as simply bumping a serial number up by one when an incompatible change is made.)

Note also:  I don't consider this a blocking-approval item; I'm just trying to point out where I think there are weaknesses in our process... although I grant that this particular one is pretty far down the list compared to others we have :)